packages/client/lib/Chain.js

```diff
@@ -187,6 +187,30 @@ export default class Chain {
     return transaction
   }
 
+  /**
+   * Create & sign a sweep transaction
+   * @param {!string} externalAddress - External Change address.
+   * @param {string[]} outputs - to, value.
+   * @param {number} [fee] - Fee price in native unit (e.g. sat/b, wei)
+   * @param {string[]} fixedInputs - txid, vout, address, value, satoshis, derivationPath
+   * @return {Promise<string>} Resolves with a signed transaction object.
+   */
+  async buildSweepTransaction (externalAddress, fee, outputs, fixedInputs) {
```

Comment:
If this is not something that is implemented cross chain, it shouldn't be included at the client level. In special situations where you need to access something on a specific chain, you can do `client.getMethod('buildSweepTransaction')` or access it directly on the provider object.

Reply:
Good point, updated in https://github.com/liquality/chainabstractionlayer/pull/358/commits/3100c572312cdd9a3510e223d7ec76ca921a060e